cmds/core/cat/cat.go

```diff
@@ -48,12 +61,6 @@ func cat(w io.Writer, files []string) error {
 func main() {
 	flag.Parse()
 
-	if flag.NArg() == 0 {
```

Comment:
I would suggest keeping arg parsing in main and calling cat as a process when you wish to do whole program testing.